### PR TITLE
Fix properties not populating particle explorer tab

### PR DESCRIPTION
**Pull request by ctrlaltdavid**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10204/HighFidelity-Beta-PR13255-8c9bdf51a7077b11347951c723d9b7e90a4fff6f.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10204/HighFidelity-Beta-Interface-PR13255-8c9bdf51a7077b11347951c723d9b7e90a4fff6f.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10204/HighFidelity-Beta-PR13255-8c9bdf51a7077b11347951c723d9b7e90a4fff6f.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10204/HighFidelity-Beta-Interface-PR13255-8c9bdf51a7077b11347951c723d9b7e90a4fff6f.exe

**Comment by qligmaloney** [comment]
Currently testing

**Comment by qligmaloney** [comment]
PASS @conklin94122 Particle explorer tab populates. All steps of test plan validated without issue.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10278/HighFidelity-Beta-PR13255-3c5a779.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10278/HighFidelity-Beta-PR13255-3c5a779.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10278/HighFidelity-Beta-Interface-PR13255-3c5a779.exe

**Comment by ctrlaltdavid** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10288/HighFidelity-Beta-PR13255-3c5a779.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10288/HighFidelity-Beta-PR13255-3c5a779.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10288/HighFidelity-Beta-Interface-PR13255-3c5a779.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10288/HighFidelity-Beta-PR13255-3c5a779.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10288/HighFidelity-Beta-Interface-PR13255-3c5a779.exe

File: scripts/system/particle_explorer/particleExplorerTool.js [hunk 1]
```diff
@@ -16,37 +16,55 @@ var PARTICLE_EXPLORER_HTML_URL = Script.resolvePath('particleExplorer.html');
...
         that.activeParticleEntity = 0;
+        that.activeParticleProperties = {};
 
         var messageData = {
             messageType: "particle_close"
         };
         that.webView.emitScriptEvent(JSON.stringify(messageData));
+    };
+
+    function sendActiveParticleProperies() {
```
[the remark this answers is not in the document]
Comment (ctrlaltdavid) [reply scripts/system/particle_explorer/particleExplorerTool.js]: Done

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10342/HighFidelity-Beta-PR13255-b4b6318.apk

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10342/HighFidelity-Beta-Interface-PR13255-b4b6318.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10342/HighFidelity-Beta-PR13255-b4b6318.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10342/HighFidelity-Beta-Interface-PR13255-b4b6318.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Fail @conklin94122 
When switching tabs and coming back to the particle, the values do not stick. We are unable to test against master and stable due to the particle tab showing no information.

Video and Log attached
[Files.zip](https://github.com/highfidelity/hifi/files/2074214/Files.zip)

**Comment by ctrlaltdavid** [comment]
Manuscript case added per new problem discovered during QA: https://highfidelity.manuscript.com/f/cases/15613

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10400/HighFidelity-Beta-PR13255-32c21a1.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10400/HighFidelity-Beta-PR13255-32c21a1.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10400/HighFidelity-Beta-Interface-PR13255-32c21a1.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10400/HighFidelity-Beta-PR13255-32c21a1.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10400/HighFidelity-Beta-Interface-PR13255-32c21a1.exe

**Comment by QliCknox** [comment]
Currently testing

**Comment by QliCknox** [comment]
PASS @conklin94122 
All steps of test plan validated without issue.

**Pull request merged**